### PR TITLE
ansible-git container files

### PR DESCRIPTION
**Pull request by andrewWithTheRedHat**
adding ansible-git container build config as docker inline build and image stream object - we can add the image stream to the build config if thats the better route.

### CODE REVIEW

File: containers/ansible/ansible-rfe-buildconfig.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  runPolicy: "Serial" 
+  source:
+    type: Dockerfile
+    dockerfile: |
+      FROM registry.access.redhat.com/ubi8/python-38
+      
+      #set to root user
+      USER 0
+      
+      #update the repos
```
Comment (sabre1041) [inline containers/ansible/ansible-rfe-buildconfig.yml:15]: Please combine run statements to consolidate layers

File: containers/ansible/ansible-rfe-imagestream.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: ansible-rfe-build
```
Comment (sabre1041) [inline containers/ansible/ansible-rfe-imagestream.yml:4]: Can we rename `ansible-rfe`

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Small changes requested

File: containers/ansible/ansible-rfe-imagestream.yml (hunk 2 above)
Comment (andrewWithTheRedHat) [reply containers/ansible/ansible-rfe-imagestream.yml]: changed

File: containers/ansible/ansible-rfe-buildconfig.yml (hunk 1 above)
Comment (andrewWithTheRedHat) [reply containers/ansible/ansible-rfe-buildconfig.yml]: changed

**Comment by andrewWithTheRedHat** [comment]
removing containers directory, adding rfe/ansible-rfe-runner directory.  shrinking buildconfig file and renaming extension to match project

**Comment by andrewWithTheRedHat** [comment]
edited dockerfile for single line run command, added kustomization file for gitops, edited kustomization for overlays

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**